script/bulk_import/vbulletin.rb

```diff
@@ -5,6 +5,44 @@
 class BulkImport::VBulletin < BulkImport::Base
 
   SUSPENDED_TILL ||= Date.new(3000, 1, 1)
+  CHARSET_MAP = {
+    "armscii8" => nil,
```

Comment:
Charset of mysql2 client and the ruby standard lib are not consistent. For example, mysql2 accept "utf8" while Ruby standard using "utf-8" (will throw an error if we put "utf8").
That's why there is CHARSET_MAP there.